client/my-sites/sidebar/test/sidebar.js

```diff
@@ -69,24 +69,7 @@ describe( 'MySitesSidebar', () => {
 			expect( wrapper.html() ).toEqual( null );
 		} );
 
-		test( 'Should return store menu item if user can use store on this site', () => {
-			const Sidebar = new MySitesSidebar( {
-				canUserUseCalypsoStore: true,
-				isSiteWpcomStore: true,
-				...defaultProps,
-				site: {
-					plan: {
-						product_slug: 'business-bundle',
-					},
-				},
-			} );
-			const Store = () => Sidebar.store();
-
-			const wrapper = shallow( <Store /> );
-			expect( wrapper.props().link ).toEqual( '/store/mysite.com' );
-		} );
-
-		test( 'Should return wp-admin menu item if user can use store on this site and the site is an ecommerce plan', () => {
+		test( 'Should return Calypsoified store menu item if user can use store on this site and the site is an ecommerce plan', () => {
```

Comment:
This test description is incorrect. Should be 'Should return wp-admin menu item if user can use store on this site and the site is an ecommerce plan'

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/49641/commits/af47a92df6c190b7d9e9a291d07b60adb3a081e8